internal-projects/FacGame.md

```diff
@@ -0,0 +1,44 @@
+## Title of project
+GunBoat Diplomacy
+
+## Screencast
```

Comment:
You can remove this heading. It will be obvious that it is a screencast

Reply:
True, thank you